docs/application/tizen-studio/release-notes/4-5-release-notes.md

```diff
@@ -0,0 +1,68 @@
+# Tizen Studio 4.5 Release Notes
+
+- Release Date: Oct 24, 2021
+
+## IDE and Tools
+
+### New Feature
+
+- IDE
+  - Tizen 6.5 platform support has been added.
```

Comment:
@manishtr would you also add `TIDL` support both on `IDE` and `CLI` into this `New Feature` section?